eclair-core/src/test/scala/fr/acinq/eclair/integration/basic/fixtures/ThreeNodesFixture.scala

```diff
@@ -0,0 +1,31 @@
+package fr.acinq.eclair.integration.basic.fixtures
+
+import akka.actor.ActorSystem
+import akka.testkit.TestKit
+import fr.acinq.eclair.NodeParams
+
+
+case class ThreeNodesFixture(system: ActorSystem,
+                             alice: MinimalNodeFixture,
+                             bob: MinimalNodeFixture,
+                             charlie: MinimalNodeFixture) {
```

Comment:
What a hostile takeover! Are we deprecating poor Carol in favor of Charlie?

Reply:
Sorry Carol! Not sure why I switched to the NATO phonetic alphabet. And it's not the first time, did the same in other tests apparently.

Fixed in 3d42e3af8.